frontend/src/app/testQueue/AoEForm/AoEForm.tsx

```diff
@@ -50,6 +27,12 @@ interface Props {
     internalId: string;
     gender: string;
   };
+  lastTest:
+    | {
+        dateTested: string;
+        result: string;
+      }
+    | undefined;
```

Comment:
This seems like a syntax error. I am unclear what unioning an non existent type does. I would remove the leading`|`
```suggestion
  lastTest: {
        dateTested: string;
        result: string;
      }
    | undefined;
```

Reply:
This is prettier auto-formatting. It just puts the leading `|` back if I remove it, and it even inserts one if I put `undefined` first in the union